pkg/controller/storagecluster/storagecluster.go

```diff
@@ -1441,6 +1441,7 @@ func (c *Controller) CreatePodTemplate(
 	podSpec.NodeName = node.Name
 	labels := c.StorageClusterSelectorLabels(cluster)
 	labels[constants.OperatorLabelManagedByKey] = constants.OperatorLabelManagedByValue
+	labels[constants.K8sNodeRunningPxPod] = node.Name
```

Comment:
I was initially thinking that we should update the label for existing pods as well (without portworx update), but looks like we will support the upgrade feature starting portworx 3.1.2/3.2.0. So users will anyway have to upgrade portworx and will end up re-creating portworx pods.

Reply:
Right so this change should go along with pxe release

Reply:
Yes, before the pxe release.